crates/driver/src/infra/liquidity/config.rs

```diff
@@ -231,3 +235,13 @@ impl BalancerV2 {
         })
     }
 }
+
+/// ZeroEx liquidity fetching options.
+#[derive(Clone, Derivative)]
+#[derivative(Debug)]
```

Comment:
Does this crate have other advantages than ignoring a field from being printed? I think for this we could also define our own secret type which wraps string and doesn't implement Display or Debug (which might be less error prone than relying on engineers remembering to add the derivative macro)

Reply:
It is also used as follows:
1. https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/database/competition.rs#L61-L62
2. https://github.com/cowprotocol/services/blob/main/crates/shared/src/zeroex_api.rs#L216-L218

> we could also define our own secret type which wraps string and doesn't implement Display or Debug

Will try this in a separate PR, but technically it's the same: don't forget to use a special type for secret fields.

Reply:
> but technically it's the same: don't forget to use a special type for secret fields.

Agree, but I think it's slightly more memorable to have a deliberate secret type than remembering to use a specific derive macro, but I don't feel strongly.